src/Store.rei

```diff
@@ -1,12 +1,12 @@
-module type Store = {
+type unsubscribe = unit => unit;
+
+module type S = {
```

Comment:
Using `S` for the "main" module types is a common convention in OCaml-land. Breaking change, but unlikely to actually affect consumers I think.

Reply:
Oh, good to know! Thanks for making this more idiomatic